cli/paas/client.go

```diff
@@ -314,6 +314,8 @@ func (c *CarrierClient) gitPush(name, tmpDir string) error {
 	cmd := exec.Command("/bin/sh", "-c", fmt.Sprintf(`
 cd "%s" 
 git init
+git config user.name "Carrier"
+git config user.email ci@carrier
```

Comment:
Since there is no validation for the domain I though a non-domain is fine. We don't have a relevant domain and it shouldn't matter much anyway.

Reply:
In the future we may wish to insert information about the user doing the push, maybe the hostname as well, if we need a domain part. That info would have to come from the OS in some way, or we check the git config in some way (is there a go package for such git ops ?)